### DISCUSSION TITLE
动态增加/删除字段问题。

### DISCUSSION THREAD

**Discussion by Lind-pro** [alibaba/formily#2701] [Q&A 2.x] [answered]
使用```createField``` 来动态追加字段 不生效
https://codesandbox.io/s/awesome-worker-k4eev?file=/App.tsx

**Comment by janryWang** [comment, marked as answer]
不要调用createField，请使用Field组件

**Reply by Lind-pro** [reply]
codesandbox 上的例子应该怎么修改呢？

**Reply by Lind-pro** [reply]
直接new Field? 好像不行。

**Reply by janryWang** [reply]
用Field组件！@formily/react导出

**Reply by Lind-pro** [reply]
> 用Field组件！@formily/react导出

 Field({
          decorator: [FormItem],
          component: [Input],
          name: "testa"
        });

这样写不对。。。。

**Comment by Lind-pro** [comment]
哦。``` <Field name="input" component={[Input, { placeholder: 'Please Input' }]} />``` 那我怎么放在Form下面去呢

**Comment by Lind-pro** [comment]
好吧，大概明白了，我这个思路应该是没法实现。。我参考文档写吧

**Comment by janryWang** [comment]
建议你看看受控表单 这个文档

**Reply by Lind-pro** [reply]
嗯，感谢大佬。https://formilyjs.org/zh-CN/guide/advanced/controlled#schema-%E5%8F%97%E6%8E%A7 这个方案应该适合我
